src/toolkits/object_detection/object_detector.cpp

```diff
@@ -281,6 +281,13 @@ void object_detector::init_options(
       "If True, print progress updates and model details.",
       true,
       true);
+  options.create_categorical_option(
+     /* name              */ "model",
+      /* description       */
+      "Defines the model type",
```

Comment:
FYI - Looks like there are some indentation issues here. I also don't think you want a new line here.

Reply:
Yeah. I just did it.

Reply:
I just did it.